pkgs/build-support/testers/default.nix

```diff
@@ -118,6 +118,7 @@
        * make sense for this evaluation of Nixpkgs.
        */
       nixosTesting =
+        lib.warn "testers.nixosTest is deprecated. Use testers.runNixOSTest instead."
```

Comment:
The ofborg failure could be fixed by moving this warning into the function body under `test:`. Non ideal that this is necessary though..

Reply:
Never mind, looks like that doesn't work either..

Thinking about this again, I believe we need to not expose the attribute at all if `config.allowAliases = false` (which then also prevents uses in Nixpkgs).